components/brave_ads/browser/ads_p2a.cc

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/ads_p2a.h"
+
+#include <stdint.h>
+
+#include "base/metrics/histogram_macros.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/values.h"
+#include "brave/components/brave_ads/common/pref_names.h"
+#include "brave/components/weekly_storage/weekly_storage.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
```

Comment:
already included in the header file

Reply:
@iefremov in `ads_p2a.h`?

Reply:
nevermind, apparently they are not in ads_p2a.h :)